WebViewControl/WebView.cs

```diff
@@ -481,7 +491,15 @@ public partial class WebView : IDisposable {
                     return;
                 }
 
-                var frameName = e.Frame.Name;
+                var frame = e.Frame;
+                var frameName = frame.Name;
+
+                OperationOccurred?.Invoke("JS context released", new Dictionary<string, string> {
+                    { "frameName", frameName },
+                    { "isValid", frame.IsValid.ToString() },
+                    { "this.IsMainFrame", this.IsMainFrame(frameName).ToString() },
+                    { "frame.IsMain", frame.IsMain.ToString() },
```

Comment:
i added these 2, because they are not 100% compatible. When creating and quickly destroying an iframe, the frame Name is empty - and it would be icorrectly considered as being the main frame